openshift/app.dc.yaml

```diff
@@ -15,21 +26,24 @@ objects:
       name: "${APP_NAME}-app-${JOB_NAME}"
     spec:
       replicas: 2
+      revisionHistoryLimit: 10
       selector:
         app: "${APP_NAME}-${JOB_NAME}"
         deploymentconfig: "${APP_NAME}-app-${JOB_NAME}"
+        role: app
       strategy:
         type: Rolling
+        resources: {}
         rollingParams:
           timeoutSeconds: 600
-          resources: {}
       template:
         metadata:
           annotations:
             openshift.io/generated-by: OpenShiftNewApp
           labels:
             app: "${APP_NAME}-${JOB_NAME}"
             deploymentconfig: "${APP_NAME}-app-${JOB_NAME}"
+            role: app
```

Comment:
Adding role app is future-proofing?  Doesn't seem like it is used just yet?

Reply:
Yes - this is recommended as part of their best practice guidelines. In this case it isn't being used as it's a singleton pod, but other projects and deployments with more complexity will need it.